src/shared/env/env.contants.ts

```diff
@@ -0,0 +1,7 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+export const ENV = Object.freeze({
+  ...process.env,
```

Comment:
It's better to extract only necessary env variables to avoid  system-wide (sensitive) data leakage